{"name": "test_close_order_card[firefox]", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:511:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16", "trace": "self = <tests.test_order_feed.TestOrderFeed object at 0x1044ea2b0>, setup_home_page = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"9f85d30a-ec70-4670-8216-1acb4a25122c\")>\n\n    def test_close_order_card(self, setup_home_page):\n        driver = setup_home_page\n        close_card = FeedPage(driver)\n>       assert close_card.close_order_card()\n\ntests/test_order_feed.py:15: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/feed_page.py:66: in close_order_card\n    self.click_on_order_card()\npages/feed_page.py:54: in click_on_order_card\n    self.click_element(self.FeedLocators.ORDER_CARDS)\npages/base_page.py:54: in click_element\n    self.find_element(locator, timeout).click()\npages/base_page.py:51: in find_element\n    return WebDriverWait(self.driver, timeout).until(EC.visibility_of_element_located(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"9f85d30a-ec70-4670-8216-1acb4a25122c\")>, method = <function visibility_of_element_located.<locals>._predicate at 0x1044f8430>, message = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       RemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nE       WebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nE       NoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:511:5\nE       dom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\n.venv/lib/python3.9/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException"}, "steps": [{"name": "Процедура открытия заказа", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:511:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\n", "trace": "  File \"/Users/milanradonich/PycharmProjects/Diplom/diplom_3/pages/feed_page.py\", line 66, in close_order_card\n    self.click_on_order_card()\n  File \"/Users/milanradonich/PycharmProjects/Diplom/diplom_3/pages/feed_page.py\", line 54, in click_on_order_card\n    self.click_element(self.FeedLocators.ORDER_CARDS)\n  File \"/Users/milanradonich/PycharmProjects/Diplom/diplom_3/pages/base_page.py\", line 54, in click_element\n    self.find_element(locator, timeout).click()\n  File \"/Users/milanradonich/PycharmProjects/Diplom/diplom_3/pages/base_page.py\", line 51, in find_element\n    return WebDriverWait(self.driver, timeout).until(EC.visibility_of_element_located(locator))\n  File \"/Users/milanradonich/PycharmProjects/Diplom/diplom_3/.venv/lib/python3.9/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Клик по кнопке «Лента заказов»", "status": "passed", "steps": [{"name": "Клик по кнопке Лента заказов", "status": "passed", "start": 1730900611539, "stop": 1730900611789}], "start": 1730900611539, "stop": 1730900611792}, {"name": "Клик по заказу", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:511:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\n", "trace": "  File \"/Users/milanradonich/PycharmProjects/Diplom/diplom_3/pages/feed_page.py\", line 54, in click_on_order_card\n    self.click_element(self.FeedLocators.ORDER_CARDS)\n  File \"/Users/milanradonich/PycharmProjects/Diplom/diplom_3/pages/base_page.py\", line 54, in click_element\n    self.find_element(locator, timeout).click()\n  File \"/Users/milanradonich/PycharmProjects/Diplom/diplom_3/pages/base_page.py\", line 51, in find_element\n    return WebDriverWait(self.driver, timeout).until(EC.visibility_of_element_located(locator))\n  File \"/Users/milanradonich/PycharmProjects/Diplom/diplom_3/.venv/lib/python3.9/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1730900611792, "stop": 1730900621980}], "start": 1730900611539, "stop": 1730900621980}], "parameters": [{"name": "driver_setup", "value": "'firefox'"}], "start": 1730900611539, "stop": 1730900621981, "uuid": "f7b9739b-1fad-4ce2-be12-0612d4d9ed9f", "historyId": "482d87f0e7dbea004a51bea38f125859", "testCaseId": "8b38e7a8d8b6d94ec631f46ae59a6107", "fullName": "tests.test_order_feed.TestOrderFeed#test_close_order_card", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_order_feed"}, {"name": "subSuite", "value": "TestOrderFeed"}, {"name": "host", "value": "MacBook-Air-Milan.local"}, {"name": "thread", "value": "87646-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_order_feed"}]}